apps/asap-server/src/autogenerated-gql/graphql.ts

```diff
@@ -1375,7 +1377,7 @@ export type Asset = {
   fileType: Scalars['String'];
   /** The version of the file. */
   fileVersion: Scalars['Int'];
-  /** The ID of the object. */
+  /** The ID of the object (usually GUID). */
   id: Scalars['String'];
```

Comment:
I would not say that.
After migration to other CMS this might change :)

Reply:
should be reverted now. `schema:update` did weird things...